.gitignore

```diff
@@ -8,6 +8,7 @@ _obj
 _test
 .cover/
 .idea/
+.vscode/
```

Comment:
You can also setup your local machine to have a `.gitignore` that's always applied to any git repo you're working in:

```sh
touch ~/.gitignore
git config --global core.excludesfile ~/.gitignore
```

And you never have to worry about a project not having `.gitignore` set up for your editor of choice.